resources/schema/general/form.schema

```diff
@@ -21,6 +21,37 @@
       "description": "The form must be completed for the project level status to advance.  If this form is on a project this can only be PROJECT.  All required fields on the form must be completed for the project status to advance.  If this form is on a station then it can be PROJECT or STATION.  If on a station and set to PROJECT then all required fields on the form must be completed for the project status to advance.  If on a station and set to STATION then all required fields on the form must be completed for the station status to advance.  This field is read only and cannot be updated through API calls.",
       "enum": ["PROJECT", "STATION"]
     },
+    "fieldDefinitions": {
```

Comment:
add ticket - how to handle switching to a client file that doesn't have a form with this name